service-api/src/main/java/greencity/dto/payment/PaymentResponseDto.java

```diff
@@ -0,0 +1,101 @@
+package greencity.dto.payment;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.*;
+import org.hibernate.validator.constraints.Length;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@ToString
+public class PaymentResponseDto {
+    @Length(max = 1024)
+    @JsonProperty("order_id")
+    private String orderId;
```

Comment:
Please revert this dto to the previous state, this is a part of logic. If you rename the fields you drop this logic.